active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+    "    # train model on train indices in this split\n",
+    "    predictor.fit(\n",
+    "        train_data=df_labeled,\n",
+    "        hyperparameters=hypterparameters,\n",
+    "    )\n",
+    "    # predicted probabilities for test split\n",
+    "    pred_labels = predictor.predict(data=df_test)\n",
+    "    true_labels_test = np.array(df_test['label'].tolist())\n",
+    "    model_accuacy_arr[i] = np.mean(pred_labels == true_labels_test)\n",
+    "    \n",
+    "    print('test round: ', i, 'accuracy: ', np.mean(pred_labels == true_labels_test))\n",
+    "        \n",
+    "    print('computing active learning scores')\n",
+    "    # compute active learning scores\n",
+    "    active_learning_scores, active_learning_scores_unlabeled = get_active_learning_scores(\n",
+    "        labels_labeled, pred_probs_labeled, pred_probs_unlabeled\n",
+    "    )\n",
+    "    \n",
+    "    print('getting idx to relabel')\n",
+    "    # get the indices of examples to collect more labels for\n",
+    "    relabel_idx_unlabeled = get_idx_to_label(\n",
+    "        active_learning_scores_unlabeled=active_learning_scores_unlabeled,\n",
+    "        batch_size_to_label=batch_size_to_label,\n",
+    "    )\n",
+    "    \n",
+    "    relabel_idx_unlabeled = np.random.choice(range(df_unlabeled.shape[0]), batch_size_to_label, replace=False)\n",
+    "    print('setting up next iter')\n",
+    "    # format the data for the next round of active learning, ie. moving some unlabeled \n",
+    "    # examples to the labeled pool because we are collecting labels for them\n",
+    "    df_labeled, df_unlabeled = setup_next_iter_data(df_labeled, df_unlabeled, relabel_idx_unlabeled)\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Initial model test accuracy: {model_accuacy_arr[0]:.3}\")\n",
+    "print(f\"Final model test accuracy (after 15 rounds of active learning): {model_accuacy_arr[-1]:.3}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.save(\"model_acc_30_rounds_activelab\", model_accuacy_arr)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.plot(model_accuacy_arr)\n",
+    "plt.xticks(range(num_rounds))\n",
+    "plt.xlabel(\"Round\")\n",
+    "plt.ylabel(\"Model Accuracy\")\n",
+    "plt.show()\n",
+    "\n",
+    "plt.savefig('model_acc_30_rounds_activelab.png')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the plot above, we can see that the model accuracy increases steadily with each additional round of collecting more labels, getting improved consensus labels, and model training."
```

Comment:
why do you mention consensus labels? there should be no estimation of consensus labels in this single-label setting